_dashboards/discover/index.md

```diff
@@ -1,17 +1,94 @@
 ---
 layout: default
-title: Discover
-nav_order: 10
+title: Exploring data with Discover
+nav_order: 20
 has_children: true
 ---
 
-# Discover 
+# Exploring data with Discover 
 
-**Discover** in OpenSearch Dashboards helps you extract insights and get value out of data assets across your organization. You can quickly ingest and query your data, display that data in visualizations and interactive dashboards, and deliver insights to your organization.
+**Discover** in OpenSearch Dashboards helps you extract insights and get value out of data assets across your organization. Discover enables you to:
 
-<img src="{{site.url}}{{site.baseurl}}/images/discover-index.png" alt="Discover user interface">
+1. **Select data.** You can explore, customize, and filter data, as well as search data using the [Dashboards Query Language (DQL)]({{site.url}}{{site.baseurl}}/dashboards/dql/).
+2. **Explore the data.** You can analyze data details, view individual documents, and create tables summarizing data contents.
+3. **Visualize data.** You can display findings in a single dashboard showing different data visualization types.
```

Comment:
I think this makes me believe that discover itself can display different visualization types.

```
You can display findings in a single dashboard showing different data visualization types.
```
We could not visualize directly in discover except bar chart for specific field. We need to save the search, then go to visualization, then select the saved search to create diff visualization types.

I think it could be less confusing if we mention that we could visualize the saved search created from discover in different data visualization types.

Reply:
Does the following revision sufficiently clarify about saved searches? 

`You can display findings from your saved searches in a single dashboard showing different data visualization types.`